lib/main.dart

```diff
@@ -1,125 +1,125 @@
+  static Movie marioBros = Movie(
+    title: 'The Super Mario Bros. Movie',
+    originalTitle: 'The Super Mario Bros. Movie',
+    thumbnail: 'images/backdrop_mario_bros_movie.jpg',
+    poster: 'images/poster_mario_bros_movie.jpg',
+    overview: 'While working underground to fix a water main, Brooklyn '
+        'plumbers—and brothers—Mario and Luigi are transported down a '
+        'mysterious pipe and wander into a magical new world. But when the '
+        'brothers are separated, Mario embarks on an epic quest to find '
+        'Luigi.',
+    voteAverage: 7.8,
+    genres: ['Animation', 'Family', 'Adventure', 'Fantasy', 'Comedy'],
+    countLikes: 5,
+  );
+
   const MyApp({super.key});
 
-  // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Flutter Demo',
-      theme: ThemeData(
-        // This is the theme of your application.
-        //
-        // TRY THIS: Try running your application with "flutter run". You'll see
-        // the application has a blue toolbar. Then, without quitting the app,
-        // try changing the seedColor in the colorScheme below to Colors.green
-        // and then invoke "hot reload" (save your changes or press the "hot
-        // reload" button in a Flutter-supported IDE, or press "r" if you used
-        // the command line to start the app).
-        //
-        // Notice that the counter didn't reset back to zero; the application
-        // state is not lost during the reload. To reset the state, use hot
-        // restart instead.
-        //
-        // This works for code too, not just values: Most code changes can be
-        // tested with just a hot reload.
-        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
-        useMaterial3: true,
+      home: Scaffold(
+        appBar: AppBar(
+          backgroundColor: Colors.black87,
+          title: Text(marioBros.originalTitle),
+        ),
+        body: SafeArea(
+          child: Center(
+            child: MovieDeck(marioBros),
+          ),
+        ),
       ),
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
     );
   }
 }
 
-class MyHomePage extends StatefulWidget {
-  const MyHomePage({super.key, required this.title});
-
-  // This widget is the home page of your application. It is stateful, meaning
-  // that it has a State object (defined below) that contains fields that affect
-  // how it looks.
-
-  // This class is the configuration for the state. It holds the values (in this
-  // case the title) provided by the parent (in this case the App widget) and
-  // used by the build method of the State. Fields in a Widget subclass are
-  // always marked "final".
-
-  final String title;
-
-  @override
-  State<MyHomePage> createState() => _MyHomePageState();
-}
-
-class _MyHomePageState extends State<MyHomePage> {
-  int _counter = 0;
+class MovieDeck extends StatelessWidget {
```

Comment:
Move this class to a separate file

Reply:
done

Reply:
Maybe you forgot to push?

Reply:
sorry, i confused movie deck with movie. Now it´s done